src/v2/Components/Artwork/Details.tsx

```diff
@@ -218,6 +233,66 @@ export const Details: React.FC<DetailsProps> = ({
   )
 }
 
+interface LotCloseInfoProps {
+  saleArtwork: any
+  sale: any
+}
+
+export const LotCloseInfo: React.FC<LotCloseInfoProps> = ({
+  saleArtwork,
+  sale,
+}) => {
+  const { hasEnded: lotHasClosed, time } = useTimer(
+    saleArtwork.endAt!,
+    sale.startAt
+  )
+
+  const { hasEnded: lotsAreClosing, hasStarted: saleHasStarted } = useTimer(
+    sale.endAt!,
+    sale.startAt!
+  )
+
+  if (!saleHasStarted) {
+    return null
+  }
+
+  const saleEndDate = sale.auctionsDetailFormattedStartDateTime
+  const timerCopy = getTimerCopy(time, saleHasStarted)
+
+  let lotCloseCopy
+
+  // Lot has already closed
+  if (lotHasClosed) {
+    lotCloseCopy = "Closed"
+  } else if (saleHasStarted) {
+    // Sale has started but lot has not started closing
+    if (lotsAreClosing) {
+      lotCloseCopy = `Closes, ${timerCopy.copy}`
+      // Sale has started but lots have not started closing
+    } else {
+      lotCloseCopy = `Closes, ${saleEndDate}`
+    }
+  }
+
+  return (
+    <Text variant="xs" color={getLabelColor(timerCopy.color, lotHasClosed)}>
+      {lotCloseCopy}
+    </Text>
+  )
+}
+
+const getLabelColor = (color, lotHasClosed): string => {
```

Comment:
shame the colors are different at the sale detail level so this can't be extracted to component and reused, but I'll definitely follow this pattern